sdk/Cargo.toml

```diff
@@ -175,6 +175,7 @@ solana-signature = { workspace = true, features = [
     "verify",
 ], optional = true }
 solana-signer = { workspace = true, optional = true }
+solana-system-transaction = { workspace = true }
```

Comment:
Ah sorry, I missed this -- the new crate should be optional and added under the "full" feature. That's why CI is failing to build `sdk` in BPF